.editorconfig

```
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 3
```

your editorconfig does not match neither the repositories style, nor common sense

```suggestion
indent_style = space
indent_size = 4
```

---

:starecry: sorry bout that, fixed now